index.d.ts

```diff
@@ -1584,6 +1584,29 @@ declare namespace Moleculer {
 			delimiter(value: string): void;
 		}
 	}
+
+	namespace Utils {
+		function isFunction(func: any): boolean;
+		function isString(str: any): boolean;
+		function isObject(obj: any): boolean;
+		function isPlainObject(obj: any): boolean;
+		function isDate(date: any): boolean;
+		function flatten(arr: any[]): any[];
+		function humanize(millis: number): string;
+		function generateToken(): string;
+		function removeFromArray(arr: any[], item: any): any[];
+		function getNodeID(): string;
+		function getIpList(): any[];
+		function isPromise(promise: any): boolean;
+		// function polyfillPromise(P: Promise): any; // From Utils -> NOT USED & NOT TESTED YET !!!
+		function clearRequireCache(filename: string): void;
+		function match(text: string, pattern: string): boolean;
+		function deprecate(prop: any, msg: string): void;
+		function safetyObject(obj: any, options: any): any;
+		function dotSet(obj: any, path: string, value: any): any;
```

Comment:
I'm cribbing a bit from lodash's types for `set` here, but its effectively the same concept.  lodash has this defined as:

`set<T extends object>(object: T, path: PropertyPath, value: any): T;`

To try to keep this somewhat simple, I think `dotSet<T extends object>(obj: T, path: string, value: unknown): T;` makes the most sense.